### PR TITLE
Feature/adding uint64 support

### PR DESCRIPTION
**Pull request by rotemtam**
this fixes issue #28

### CODE REVIEW

**Comment by mike9005** [comment]
@rotemtam thanks for taking care of this! someone on our team will look at this and hopefully we can get it out soon 🙌

**Comment by rotemtam** [comment]
@mike9005 ✅

File: README.md [hunk 1]
```diff
@@ -64,6 +64,7 @@ key.converter=com.blueapron.connect.protobuf.ProtobufConverter
 | ------------------ | ----------- |
 |Timestamp           |Timestamp    |
 |Date                |Date         |
+|Decimal             |uint64       |
```
Comment (bayardneville) [inline README.md]: Is this accurate? It seems this PR only adds uint64 to `To Connect`.

**Comment by mike9005** [comment]
@rotemtam Thanks again - we'll push this in the next release to Maven and let you know when that's done.

**Comment by ofirski** [comment]
@mike9005 thanks for merging this change.
When should we expect the next release to Maven?

**Comment by mike9005** [comment]
@ofirski we'll roll this and #32 into the next release when folks on our team are back from vacation in a week or two - will make sure we ping you to let you know when it's available.

**Comment by ofirski** [comment]
Thanks @mike9005

**Comment by bayardneville** [comment]
@rotemtam @ofirski this is included in the [2.3.0 release](https://github.com/blueapron/kafka-connect-protobuf-converter/releases/tag/v2.3.0) and is available in [Maven Central](https://search.maven.org/artifact/com.blueapron/kafka-connect-protobuf-converter/2.3.0/jar). Thanks for the contribution.

**Comment by ofirski** [comment]
Great news, thanks for the update @bayardneville.

**Pull request merged**